octopoes/octopoes/api/router.py

```diff
@@ -524,7 +527,38 @@ def recalculate_bits(octopoes: OctopoesService = Depends(octopoes_service)) -> i
 
     octopoes.commit()
 
-    return inference_count
+    RECALCULATE_BITS_SESSIONS[session_id] = inference_count
+
+
+@router.post("/bits/recalculate", tags=["Bits"])
+async def recalculate_bits(octopoes: OctopoesService = Depends(octopoes_service)) -> Any:
+    session_id = uuid.uuid4()
+    RECALCULATE_BITS_SESSIONS[session_id] = -1
+
+    asyncio.create_task(recalculate_bits_launcher(octopoes, session_id))
+
+    return {"session_id": session_id}
+
+
+@router.post("/bits/recalculate/status", tags=["Bits"])
```

Comment:
Perhaps a GET request makes more sense?

Reply:
I opted for a POST for two reasons:
1. To be consistent with the current recalculate bits implementation
1. So that we can pass parameters in the body, not in the URL

I'm cool either way, but this implementation works and I do not seem  the harm in it.

Reply:
Right but by the same argument, we are also changing the internal state of the dictionary (and later XTDB) on success... anyway... I will yield to the GET.